common/djangoapps/student/auth.py

```diff
@@ -105,6 +105,19 @@ def get_user_permissions(user, course_key, org=None):
     return STUDIO_NO_PERMISSIONS
 
 
+def is_course_org_in_site_filter_orgs(course_key):
+    """
+    Check if the course organization is in the current site's organization filters.
+
+    :param course_key: a CourseKey object representing the course
+    :return: True if the course organization is in the site organization filters, False otherwise
+    """
+    site_orgs = configuration_helpers.get_current_site_orgs()
+    if course_key.org not in site_orgs:
+        return False
+    return True
+
+
```

Comment:
Isn't this function duplicate? Can you define a single function in EDLY app and import from there?

Reply:
As we discussed, I have changed it to use the `is_course_org_same_as_site_org` from edly app